server/utils/placements/index.test.ts

```diff
@@ -249,8 +249,37 @@ describe('placementUtils', () => {
             key: { text: 'Departure time' },
             value: { text: DateFormats.timeFromDate(DateFormats.isoToDateObj(placement.actualDepartureDate)) },
           },
-          { key: { text: 'Departure reason' }, value: { text: placement.departureReason?.name } },
-          { key: { text: 'Move on' }, value: { text: placement.departureMoveOnCategory?.name } },
+          { key: { text: 'Departure reason' }, value: { text: placement.departure?.parentReason?.name } },
+          { key: { text: 'Breach or recall' }, value: { text: placement.departure?.reason?.name } },
+          { key: { text: 'Move on' }, value: { text: placement.departure?.moveOnCategory?.name } },
+          { key: { text: 'More information' }, value: { text: placement.departure?.notes } },
+        ],
+      })
+    })
+
+    it('should return the departure information for non-Breach or recall departure', () => {
```

Comment:
There should also be a separate test for when the reason selected is 'Planned move-on', which then should show the move-on category.